src/helpers/markdown.js

```diff
@@ -91,5 +91,12 @@ export const getTableOfContent = (source) => {
   const tokens = marked.lexer(source)
   // remove the first 2 tokens to avoid meta information
   // and then return every tokens that is of header type
-  return tokens.slice(2).filter(token => token.type === "heading")
+  const headers = tokens.slice(2).filter((token) => token.type === "heading")
```

Comment:
Also, I might not want to show every level of headings in the TOC. So, better to limit to level 4

CC @grgprarup
```suggestion
  const headers = tokens.slice(2).filter((token) => token.type === "heading" && token.depth <= 4)
```